batch/batch/worker/worker.py

```diff
@@ -3017,6 +3017,7 @@ def __init__(self):
         self._jvmpools_by_cores: Dict[int, JVMPool] = {n_cores: JVMPool(n_cores, self) for n_cores in (1, 2, 4, 8)}
         self._waiting_for_jvm_with_n_cores: asyncio.Queue[int] = asyncio.Queue()
         self._jvm_initializer_task = asyncio.create_task(self._initialize_jvms())
+        self._jvm_initializer_task.add_done_callback(self._initialize_jvms_postmortem)
```

Comment:
Any exceptions could be reported via code in `shutdown()` below instead of a callback, but it seems useful to log the problem immediately rather than wait for potentially hours or more for the worker to finally be shutting down.